### ISSUE TITLE
Terminal view not usable when there are many logs

### ISSUE THREAD

**Issue by xrow** [cockpit-project/cockpit#4685]
Hi,

we had a solr pod running. That pod had problems and was creating much log output (filled with long long JAVA stack traces). The problem though was that we couldn`t use the terminal window to debug since there was so much continous logging that the browser frooze a bit all the time and the terminal window was inaccessable.

<img width="604" alt="cockpit not useable" src="https://cloud.githubusercontent.com/assets/295491/16652690/82a20436-444c-11e6-9e5e-cc436ae21d1b.PNG">

How about a stop button for the log window?

**Comment by stefwalter** [comment]
Good point. On the off chance: Do you have a pod/container that exhibits this behavior? That would make it easier for someone to get started and develop a solution.

**Comment by xrow** [comment]
Try this... If you mount "data" (the directory where the cores reside) to some empty dir, it shall fail wiht plenty of logs.

{code}
apiVersion: v1
kind: ReplicationController
metadata:
  name: solr
  labels:
    name: solr
    tier: frontend
spec:
  replicas: 1
  template:
    metadata:
      labels:
        name: solr
        tier: frontend
    spec:
      containers:
      - name: solr
        image: xrowgmbh/solr:1.0.1
        volumeMounts:
          - mountPath: /sys/fs/cgroup
            name: cgroup
            readOnly: true
          - name: data
# For testing / copy cores mount somewhere else
            mountPath: /mnt
# Step 1.) kubectl exec $(php /root/scripts/check_cron.php solr prod) --namespace=prod -it  bash
# Step 2.) cp -R /opt/solr-4.10.4/example/solr/ezp-default/data/* /mnt
            mountPath: /opt/solr/example/solr/ezp-default/data
        securityContext:
            capabilities:
              add: [SYS_ADMIN]
        ports:
          - containerPort: 8983
      volumes:
        - name: cgroup
          hostPath:
            path: "/sys/fs/cgroup"
        - name: data
          nfs:
            server: 192.168.255.41
            path: /storage/cores
            readOnly: false
{code}

**Comment by larskarlitski** [comment]
I can't imagine how we'd fix this on the browser side. Probably rate-limiting the tty channel in the bridge is the best option.

@stefwalter do we already have a way to rate-limit channels in the bridge? If not, do you think it makes sense to add it?
